src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int FIGURE_COUNT = 6;
 
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURE_COUNT];
+        for (int i = 0; i < figures.length / 2; i++) {
+            figures[i] = new FigureSupplier().getRandomFigure();
```

Comment:
Don’t create several instances of the same class if you can use only one instance for your purposes
Bad example:
```
public class Main {
  public static void main(String[] args) {
      String[] emails = {"myEmail@gmail.com", "not@Valid@.g.com"};
      for(String email : emails) {
          if (new EmailValidator().isValid(email)) {
              System.out.println("Email " + email + " is valid");
          }
      }
  }
}
```
Improved example:
```
public class Main {
  public static void main(String[] args) {
      String[] emails = {"myEmail@gmail.com", "not@Valid@.g.com"};
      EmailValidator emailValidator = new EmailValidator();
      for(String email : emails) {
          if (emailValidator.isValid(email)) {
              System.out.println("Email " + email + " is valid");
          }
      }
  }
}

```